min-turnup/phase2/ansible/playbooks/roles/master/templates/kube-apiserver.json.j2

```diff
@@ -0,0 +1,102 @@
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "kube-apiserver",
+    "namespace": "kube-system",
+    "labels": {
+      "tier": "control-plane",
+      "component": "kube-apiserver"
+    }
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-apiserver",
+        "image": "{{ phase2['docker_registry'] }}/hyperkube-amd64:{{ phase2['kubernetes_version'] }}",
+        "resources": {
+          "requests": {
+            "cpu": "250m"
+          }
+        },
+        "command": [
+             "/hyperkube",
+              "apiserver",
+              "--address=127.0.0.1",
+              "--etcd-servers=http://127.0.0.1:2379",
+              "--cloud-provider={{ phase1['cloud_provider'] }}",
+{% if phase1['cloud_provider'] == "azure" %}
+              "--cloud-config=/etc/kubernetes/azure.json",
```

Comment:
It's not so much that we can't possible get it to run on these os'es. It's more about the size of the dependency. python:2.7-slim was at 200MB when I looked. This is larger then the GCI vm image. This hits us on time to boot and is prohibitive. So now we have code like this https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L813-L845. There's a benifit to sharing templates of some form for the kubernetes components but I don't think that it's long term viable to do this with jinja